docs/content/developers/ontwerpkeuzes-informatiemodellen.md

```diff
@@ -1,2 +1,15 @@
-Bij het ontwikkelen van de API's aam de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie hier) als consequentie hebbe dat een informatiemodel aangpast moet worden. 
+## Aanpassingen RGBZ en ImZTC
+
+Bij het ontwikkelen van de API's aan de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie [hier](https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/content/developers/design-keuzes.md)) als consequentie hebben dat een informatiemodel aangepast moet worden. 
 Hieronder vermelden we de gemaakte keuzes tot aanpassing van RGBZ en ImZTC. 
+
+[nog uit te werken:]
+- Einddatum van zaak als afgeleid gegeven (en STATUSTYPE in ImZTC met hoogste volgnummer is laatste status). 
+- Omschrijving- en Toelichting-attributen niet meer AN1000 maar AN ('string'); ook BESLUIT.Publicatietekst
+- Richting van relatie ZAAK - INFORMATIEOBJECT omdraaien
+- Richting van relatie BESLUIT - INFORMATIEOBJECT omdraaien 
+- Formele historie op relaties van BESLUIT: ze kunnen gewijzigd worden
+- BESLUIT kan vastgelegd zijn als: regel dat het info-object aan dezelfde zaak gekoppeld moet zijn als het Besluit (nog niet in API)
```

Comment:
dit wil ik nog even bespreken, vanuit de techniek is deze relatie vastleggen namelijk geen wens, dus ik zie niet goed waar dit vandaan komt

Reply:
Zie dit comment: https://github.com/VNG-Realisatie/gemma-zaken/pull/362#issuecomment-434357938